nifi-api/pom.xml

```diff
@@ -21,5 +21,12 @@
         <version>0.7.0-SNAPSHOT</version>
     </parent>
     <artifactId>nifi-api</artifactId>
-    <packaging>jar</packaging>    
+    <packaging>jar</packaging>
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-properties</artifactId>
+            <version>${project.version}</version>
```

Comment:
we have avoided usage of ${project.version} references as it has caused problems with the release plugin/process.  Using explicit version if necessary is sufficient as the plugin automatically handles this.